.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: test
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    name: Build and Test
+    runs-on: ubuntu-latest
```

Comment:
You should fix that to a specific version

Reply:
ok, will do

Reply:
done